multilib/fakepo.py

```diff
@@ -4,13 +4,18 @@ class FakePackageObject(object):
     framework herein
     """
     def __init__(self, po=None, d=None):
-        # FPO's can be created from yum Package Objects or dictionaries
+        # FPO's can be created from yum/dnf Package Objects or dictionaries
         if po:
             self.name = po.name
             self.arch = po.arch
             self.provides = po.provides
             self.requires = po.requires
-            self.files = po.returnFileEntries()
+            try:
+                # yum
+                self.files = po.returnFileEntries()
+            except AttributeError:
+                # dnf
+                self.files = po.files
```

Comment:
Problem is that input of multilib is either package object or dictionary. It doesn't make sense to add config option, because then you will always have to pass parameter which describes what you actually pass.